packages/nextjs/utils/scaffold-stark/networks.ts

```diff
@@ -28,7 +28,7 @@ export function getBlockExplorerTxLink(network: string, txnHash: string) {
 
   const targetChainArr = chainNames.filter((chainName) => {
     const wagmiChain = chains[chainName as keyof typeof chains];
-    return wagmiChain.network === network;
+    return (wagmiChain as { network: string }).network === network;
```

Comment:
can change the import way and update the `chains.Chain` to `Chain` in this file
```
// import * as chains from "@starknet-react/chains";
import { devnet, sepolia, mainnet, Chain } from "@starknet-react/chains";
export const chains = {
  devnet,
  sepolia,
  mainnet,
};
```

Reply:
Great, thank you very much for the contribution.